intellij_platform_sdk/BUILD

```diff
@@ -327,6 +327,26 @@ java_library(
     ),
 )
 
+java_library(
+    name = "failureaccess",
+    testonly = True,
+    exports = select_from_plugin_api_directory(
+        android_studio = [":failureaccess"],
+        clion = [":failureaccess"],
+        intellij = [":failureaccess"],
+    ),
+)
+
+java_library(
+    name = "failureaccess_for_external_binaries",
+    visibility = ["//aspect/tools:__pkg__"],
+    exports = select_from_plugin_api_directory(
+        android_studio = [":failureaccess"],
+        clion = [":failureaccess"],
+        intellij = [":failureaccess"],
+    ),
+)
+
```

Comment:
This probably isn't correct as only the newer versions have this jar, but I'm not sure which ones.